modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessor.java

```diff
@@ -113,6 +113,15 @@ else if (ddmFormInstanceReportEvent.equals(
 							currentDDMFormInstanceRecord.
 								getFormInstanceRecordId());
 			};
+
+			Stream<DDMFormInstanceRecord> stream = streamSupplier.get();
```

Comment:
@marcosapmf, please re-use variable `stream` in all occurrences of `streamSupplier.get()`.

cc @carolmariaabb

Reply:
@marcosapmf, once you perform the change mentioned above, please trigger `ci:forward`. Thanks. 🙂

Reply:
@natocesarrego, If I use `stream` in the others occurrences this error appears: **Stream has already been operated upon or closed**. This occurs because a `Stream` object should be operated only once. Please, see [this](https://www.baeldung.com/java-stream-operated-upon-or-closed-exception) for more info about this problem.

Reply:
Nice job, @marcosapmf! 😉 I just learned from this pull. 🙂